components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -97,6 +98,7 @@ object WebExtensionSupport {
      * @param onUpdatePermissionRequest (optional) Invoked when a web extension has changed its
      * permissions while trying to update to a new version. This requires user interaction as
      * the updated extension will not be installed, until the user grants the new permissions.
+     * @param onExtensionsLoaded (optional) callback invoked when the extensions are loaded by the engine but the UI doesn't have to be initialized.
```

Comment:
nit: (optional) callback invoked when the extensions are loaded by the engine. Note that the UI (browser/page actions etc.) may not be initialized at this point.

Reply:
👍